packages/plugin-ext/src/plugin/languages/lens.ts

```diff
@@ -89,4 +89,15 @@ export class CodeLensAdapter {
         symbol.command = this.commands.converter.toSafeCommand(newLens.command ? newLens.command : CodeLensAdapter.BAD_CMD, disposables);
         return symbol;
     }
+
+    releaseCodeLenses(ids: number[]): void {
```

Comment:
Please remove `// TODO: invalidate caches and dispose command handlers` comment from above

Reply:
Done